src/DiffEqJump.jl

```diff
@@ -63,6 +63,8 @@ export RSSACR, RDirect
 
 export get_num_majumps, needs_depgraph, needs_vartojumps_map
 
+export spec_to_dep_rxs_map, rxs_to_dep_spec_map, make_dependency_graph
```

Comment:
Hmm, I'm not sure we should export these, especially since they work only on MassActionJumps. Is there a reason you added this? You can always call them through DiffEqJump.methodname if needed in a test script.

Reply:
Okay, I'll delete them. The reason I thought exporting them would be good is that it makes the user's life easier when dealing with massaction networks.